lib/alephant/broker/response/batch.rb

```diff
@@ -9,20 +9,26 @@ class Batch < Base
 
         attr_reader :components, :batch_id
 
-        def initialize(components, batch_id)
+        def initialize(components, batch_id, env)
```

Comment:
Just for consistency, maybe settle on `env` or `request_env` everywhere - i'd probably go for `request_env`